src/lib/tutorial-from-url.js

```diff
@@ -36,15 +36,10 @@ const getDeckIdFromUrlId = urlId => {
  */
 const detectTutorialId = () => {
     const queryParams = queryString.parse(location.search);
-    const tutorialID = Number(
-        Array.isArray(queryParams.tutorial) ?
-            queryParams.tutorial[0] :
-            queryParams.tutorial
-    );
-    if (!isNaN(tutorialID)) {
-        return getDeckIdFromUrlId(tutorialID);
-    }
-    return null;
+    const tutorialID = Array.isArray(queryParams.tutorial) ?
+        queryParams.tutorial[0] :
+        queryParams.tutorial;
+    return getDeckIdFromUrlId(tutorialID);
```

Comment:
Should we also add an early exit in getDeckIdFromUrlId if tutorialID is `undefined`...?

Reply:
Would a tutorial that is missing the 'urlId' field get matched to this tutorialId in this case?

Reply:
Good catch, we should exit early!